src/loading.jl

```diff
@@ -77,9 +81,9 @@ however, attempting to copy a non-zero array to an inactive parameter will throw
 Likewise, copying a `src` value of `false` to any `dst` array is valid,
 but copying a `src` value of `true` will error.
 """
-function loadmodel!(dst, src; cache = Base.IdSet())
-  ldsts, _ = functor(dst)
-  lsrcs, _ = functor(src)
+function loadmodel!(dst, src; filter = Returns(true), cache = Base.IdSet())
```

Comment:
`Returns` is not available on `1.6`. We'd need either to change to 
```suggestion
function loadmodel!(dst, src; filter = _ -> true, cache = Base.IdSet())
```
Or use Compat (which may be useful elsewhere?)